cds_migrator_kit/config.py

```diff
@@ -0,0 +1,156 @@
+
+# Assets
+# ======
+#: Static files collection method (defaults to copying files).
+COLLECT_STORAGE = 'flask_collect.storage.file'
+
+# Accounts
+# ========
+#: Email address used as sender of account registration emails.
+SECURITY_EMAIL_SENDER = SUPPORT_EMAIL
+#: Email subject for account registration emails.
+SECURITY_EMAIL_SUBJECT_REGISTER = _(
+    "Welcome to cds-migrator-kit!")
+#: Redis session storage URL.
+ACCOUNTS_SESSION_REDIS_URL = 'redis://localhost:6379/1'
+
+# Celery configuration
+# ====================
+
+BROKER_URL = 'amqp://guest:guest@localhost:5672/'
+#: URL of message broker for Celery (default is RabbitMQ).
+CELERY_BROKER_URL = 'amqp://guest:guest@localhost:5672/'
+#: URL of backend for result storage (default is Redis).
+CELERY_RESULT_BACKEND = 'redis://localhost:6379/2'
+#: Scheduled tasks configuration (aka cronjobs).
+CELERY_BEAT_SCHEDULE = {
+    'indexer': {
+        'task': 'invenio_indexer.tasks.process_bulk_queue',
+        'schedule': timedelta(minutes=5),
+    },
+    'accounts': {
+        'task': 'invenio_accounts.tasks.clean_session_table',
+        'schedule': timedelta(minutes=60),
+    },
+}
+
+# Database
+# ========
+#: Database URI including user and password
+# SQLALCHEMY_DATABASE_URI = \
+#     'postgresql+psycopg2://cds-migrator-kit:' \
+#     'cds-migrator-kit@localhost/cds-migrator-kit'
+
+# JSONSchemas
+# ===========
+#: Hostname used in URLs for local JSONSchemas.
+JSONSCHEMAS_HOST = 'cds-migrator-kit.com'
+
+# Flask configuration
+# ===================
+# See details on
+# http://flask.pocoo.org/docs/0.12/config/#builtin-configuration-values
+
+#: Secret key - each installation (dev, production, ...) needs a separate key.
+#: It should be changed before deploying.
+SECRET_KEY = 'CHANGE_ME'
+#: Max upload size for form data via application/mulitpart-formdata.
+MAX_CONTENT_LENGTH = 100 * 1024 * 1024  # 100 MiB
+#: Sets cookie with the secure flag by default
+SESSION_COOKIE_SECURE = True
+#: Since HAProxy and Nginx route all requests no matter the host header
+#: provided, the allowed hosts variable is set to localhost. In production it
+#: should be set to the correct host and it is strongly recommended to only
+#: route correct hosts to the application.
+APP_ALLOWED_HOSTS = ['localhost', '127.0.0.1']
+
+# OAI-PMH
+# =======
+OAISERVER_ID_PREFIX = 'oai:cds-migrator-kit.com:'
+
+# Debug
+# =====
+# Flask-DebugToolbar is by default enabled when the application is running in
+# debug mode. More configuration options are available at
+# https://flask-debugtoolbar.readthedocs.io/en/latest/#configuration
+
+#: Switches off incept of redirects by Flask-DebugToolbar.
+DEBUG_TB_INTERCEPT_REDIRECTS = False
+
+
+CDS_MIGRATOR_KIT_BASE_TEMPLATE = 'cds_migrator_kit/base.html'
+MIGRATION_DUMP_PATH = '/eos/media/cds/test/books/migration/records/tmp/'
+MIGRATION_LOGS_PATH = '/eos/media/cds/test/books/migration/records/tmp/logs/'
+
+MIGRATION_LOG_FILE = '{0}stats.json'.format(MIGRATION_LOGS_PATH)
```

Comment:
I would recommend to use `os.path.join`, because in my case I did a mistake and I did not put the last `/` in `MIGRATION_LOGS_PATH`.